isteven-multi-select.js

```diff
@@ -34,6 +34,14 @@ import 'angular-vs-repeat';
 
 ('use strict');
 
+/* angular.copy performance is terrible on large arrays and objects,
+ * parse and stringify JSON are native code functions which are way faster
+ * as long as what you're parsing are basic JSON data types
+ */
+function fastCopy(obj) {
+  return JSON.parse(JSON.stringify(obj));
```

Comment:
I made sure we only ever pass in proper JSON data types to this so we avoid all the potential pitfalls (Objects that are not JSON literals, dates, undefined values, etc) and feel this is an acceptable improvement.